docs/resources/service_principal.md

```diff
@@ -3,7 +3,9 @@ subcategory: "Security"
 ---
 # databricks_service_principal Resource
 
-Directly manage [Service Principals](https://docs.databricks.com/administration-guide/users-groups/service-principals.html) that could be added to [databricks_group](group.md) within workspace.
+Directly manage [Service Principals](https://docs.databricks.com/administration-guide/users-groups/service-principals.html) that could be added to [databricks_group](group.md) in Databricks workspace or account.
+
+To create service principals in the Databricks account, the provider must be configured with `host = "https://accounts.azuredatabricks.net"` on AWS deployments or `host = "https://accounts.azuredatabricks.net"` and `auth_type  = "azure-cli"` on Azure deployments
```

Comment:
is it really available for User Principals only? Is it the same SPN OBO flow issue? :(

Reply:
🤦  - corrected it to AAD tokens